v1/customer/customer.gunk

```diff
@@ -893,4 +1011,346 @@ type CustomerService interface {
 	//         },
 	// }
 	GetTaxResidenceOfCustomer(GetTaxResidenceOfCustomerRequest) GetTaxResidenceOfCustomerResponse
+
+	// UpdateCustomerAddress
+	//
+	// +gunk http.Match{
+	//         Method: "PUT",
+	//         Path:   "/v1/banks/{BankID}/customers/{CustomerID}/addresses/{CustomerAddressID}",
+	//         Body:   "*",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Customer"},
+	//         Description: "Update customer addresses information.",
+	//         Summary:     "Update customer addresses.",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/customerUpdateCustomerAddressResult",
+	//                                 },
+	//                         },
+	//                 },
+	//                 "404": openapiv2.Response{
+	//                         Description: "Returned when the resource is not found.",
+	//                 },
+	//         },
+	//         Security: []openapiv2.SecurityRequirement{
+	//                 {
+	//                         SecurityRequirement: map[string]openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                 "OAuth2": openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                         Scope: []string{
+	//                                                 "write",
+	//                                         },
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	// }
+	UpdateCustomerAddress(UpdateCustomerAddressRequest) UpdateCustomerAddressResponse
```

Comment:
Ref: customerUpdateCustomerAddressResult
Actual Return: UpdateCustomerAddressResponse

Update the reference with actual return type

Reply:
whooppss, thank you